.cspell.json

```diff
@@ -196,6 +197,7 @@
     "microservice",
     "minipass",
     "minizlib",
+    "mkdocs",
```

Comment:
Add the missing word mkdocs which was causing the MegaLinter cspell step to fail.
https://github.com/tylervz/npm-groovy-lint/actions/runs/4949858452/jobs/8852638177#step:4:285

```text
10/118 ./d43a15a7-acf3-424b-84e9-01e78274fdb2-megalinter_file_names_cspell.txt 222.99ms X
  /github/workspace/d43a15a7-acf3-424b-84e9-01e78274fdb2-megalinter_file_names_cspell.txt:101:19    - Unknown word (mkdocs)     -- github workspace mkdocs
  	 Suggestions: [medics, mdoc, docs, medco, Medco]
```